Dockerfile.dapper

```diff
@@ -15,7 +15,7 @@ RUN apt update && \
     apt install -y bash git gcc docker.io vim less file curl wget ca-certificates qemu-utils
 
 ## install golangci
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.63.4
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.64.8
```

Comment:
nit: we can always use the latest golangci, as it's backward compatible and built with the latest Go runtime version. 

https://github.com/harvester/harvester/blob/master/Dockerfile.dapper#L48

Reply:
we have seen random ci failures in the past when using latest, which is the reason for pinning this to a specific version

Reply:
Bump to latest would involve many linter errors. Consider the v1.6 schedule; we should move this stuff to v1.7.0